lib/stormpath-sdk/resource/organization_account_store_mapping.rb

```diff
@@ -0,0 +1,3 @@
+class Stormpath::Resource::OrganizationAccountStoreMapping < Stormpath::Resource::Instance
+  
```

Comment:
Where are the properties of this resource? This would look a lot like the `Stormpath::Resource::AccountStoreMapping` class.